### PR TITLE
added wrap errors #23

### PR DESCRIPTION
**Pull request by marti1125**
* I just added a some error.Wrap

please check

### CODE REVIEW

**Comment by sah4ez** [comment]
O! Cool! Thank you! You can continue with other commands. And be careful with cause of return error.

**Comment by marti1125** [comment]
hi @sah4ez I just added more errors.Wrap

File: cmd/cli/group.go [hunk 1]
```diff
@@ -150,13 +151,13 @@ func FinishGroup() cli.Command {
...
 						fmt.Println("start-join-group load error: ", err.Error())
-						return err
+						return errors.Wrap(err, "start-join-group load")
 					}
 					dh := keys.Secret(priv, pub)
 					err = api.Publish(name+intermediate, dh[:])
 					if err != nil {
 						fmt.Println("start-join-group publish error: ", err.Error())
-						return err
+						return errors.Wrap(err, "start-join-group publish")
```
Comment (sah4ez) [inline cmd/cli/group.go]: This FinishGroup method.

File: cmd/cli/encrypt.go [hunk 2]
```diff
@@ -37,22 +38,22 @@ func Encrypt() cli.Command {
 
 			priv, err := utils.Read(path, "key.bin")
 			if err != nil {
-				return err
+				return errors.Wrap(err, "can not read the path")
```
Comment (sah4ez) [inline cmd/cli/encrypt.go]: This error about failed read the key.bin

File: cmd/cli/group.go [hunk 3]
```diff
@@ -150,13 +151,13 @@ func FinishGroup() cli.Command {
 					pub, err := api.Load(intermediate)
 					if err != nil {
 						fmt.Println("start-join-group load error: ", err.Error())
-						return err
+						return errors.Wrap(err, "start-join-group load")
```
Comment (sah4ez) [inline cmd/cli/group.go]: This FinishGroup method.

**Review by sah4ez — CHANGES_REQUESTED** [review]
Need fix text of some errors and we can merge it.

**Comment by marti1125** [comment]
Hi @sah4ez I updated the error message :+1:

**Review by sah4ez — APPROVED** [review]

**Review by sah4ez — APPROVED** [review]

**Pull request merged**